ledger/catchpointtracker_test.go

```diff
@@ -401,6 +402,7 @@ func TestReproducibleCatchpointLabels(t *testing.T) {
 	defer ml.Close()
 
 	cfg := config.GetDefaultLocal()
+	cfg.MaxAcctLookback = 2
```

Comment:
I set it to explicit value to be sure it is less than CatchpointInterval or CatchpointLookback